src/network/services/force-update.ts

```diff
@@ -0,0 +1,46 @@
+import { Linking, Platform } from 'react-native';
```

Comment:
What about moving this file to the `hooks` folder?
Even though the Firebase SDK should be making HTTP requests to its server I believe that `network/services` folder is meant for this kind of **[services](https://github.com/rootstrap/react-native-base/blob/main/src/network/services/example-services.ts)**

Reply:
In which level should it be the ` hooks`  folder, inside ` network`  or inside `src`?

Reply:
Should be in `src/hooks/`, I checked and the base doesn't have that folder, you will have to create it and add your custom hook there.